Resources/Catalog/FileCatalog.py

```diff
@@ -300,50 +314,53 @@ def removeCatalog( self, catalogName ):
 
   def _getSelectedCatalogs( self, desiredCatalogs ):
     for catalogName in desiredCatalogs:
-      res = self._getCatalogConfigDetails( catalogName )
-      if not res['OK']:
-        return res
-      catalogConfig = res['Value']
-      res = self._generateCatalogObject( catalogName )
-      if not res['OK']:
-        return res
-      oCatalog = res['Value']
-      self.readCatalogs.append( ( catalogName, oCatalog, True ) )
-      self.writeCatalogs.append( ( catalogName, oCatalog, True ) )
+      result = self._getCatalogConfigDetails( catalogName )
+      if not result['OK']:
+        return result
+      catalogConfig = result['Value']
+      result = self._generateCatalogObject( catalogName )
+      if not result['OK']:
+        return result
+      oCatalog = result['Value']
+      self.readCatalogs.append( ( catalogName, oCatalog, catalogConfig['Master'] ) )
```

Comment:
For active/inactive and read/write, one should stay consistent with the CS as well no ?